.github/workflows/ci.js.yml

```diff
@@ -10,26 +10,15 @@ on:
     branches: ["master"]
 
 jobs:
-  build:
+  run-ci:
     runs-on: ubuntu-latest
-
-    strategy:
-      matrix:
-        node-version: [16.x]
```

Comment:
why removed?

Reply:
because the only one version of node is used

Reply:
What happens when we want to add the second one? Matrix strategy would be added again? 
Also why the only one is used? According to https://nodejs.org/en/about/releases/ v14 is still maintained and v18 is in current status

Reply:
Since there are 3 actual node versions developers of tokenomics could have any of them on their laptops.
Good to know that our tests work on all of these versions.

And also wrapping tests to docker is overhead in this case IMO

Reply:
hardhat is incompatible with node v18 and i don't see the point testing Maintenance LTS for the new application. let's just stick to Active LTS and specify it in package.json so developers will know which version to use

Reply:
Maintenance LTS is not deprecated. Also hardhat could start support 18 somewhen in the middle of "current" status of 18

again: my main point that we should leave settings for matrix of versions (even if we have only one). It would be convenient to change version (or versions) in future without touching unrelated lines of config

Reply:
ok